source/site-administrators/engine/engine-headers-based-authentication.rst

```diff
@@ -169,8 +169,8 @@ To add optional attributes, add the following inside the ``<headers>`` tag:
           <!-- Optional attribute mappings, allows to expose attributes from the external auth -->
           <attributes>
             <attribute>
-              <name>APP_ATTRIBUTE_NAME</name> <!-- The name of the attribute in the header -->
```

Comment:
Other places of the guide also use the `APP_`  prefix and all uppercase, I think the idea was to distinguish them as values coming from the external system. I don't think it's a good idea to change only one of the examples.

Reply:
That works. I would like the comment to remind the reader that the prefix is not needed here.